### PR TITLE
feat: adds node cl v6 examples

### PR DESCRIPTION
**Pull request by Justintime50**
Adds the Node v6 CL updated examples.

Various corrections were made to previous Node snippets such as update a webhook which never even called the update function.

Adds missing `type` param to create report examples in other langs

### CODE REVIEW

File: official/docs/node/current/batches/add-shipments.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
+const EasyPostClient = require('@easypost/api');
 
-const api = new Easypost(process.env.EASYPOST_API_KEY);
+const client = new EasyPostClient(process.env.EASYPOST_API_KEY);
 
-api.Batch.retrieve('batch_...').then((batch) =>
-  batch.addShipments(['shp_...', 'shp_...']).then(console.log),
-);
+(async () => {
+  const batch = await client.Batch.retrieve('batch_...');
```
Comment (jchen293) [inline official/docs/node/current/batches/add-shipments.js:6]: Nit: This step is redundant now as the user can just pass in the `id` string in the first parameter. This applies to other places.

File: official/docs/node/current/users/retrieve.js [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+(async () => {
+  let user;
 
-// Retrieve a child user
-api.User.retrieve('user_...').then(console.log);
+  // Retrieve the authenticated user
+  user = await client.User.retrieveMe().then(console.log);
+
+  // Retrieve a child user
+  user = await client.User.retrieve('user_...').then(console.log);
```
Comment (jchen293) [inline official/docs/node/current/users/retrieve.js:12]:
```suggestion
  user = await client.User.retrieveMe();

  // Retrieve a child user
  user = await client.User.retrieve('user_...');
```

File: official/docs/node/current/batches/add-shipments.js (hunk 1 above)
Comment (Justintime50) [reply official/docs/node/current/batches/add-shipments.js:6]: True, though we seem to do this everywhere so I'll keep it.

File: official/docs/node/current/users/retrieve.js (hunk 2 above)
Comment (Justintime50) [reply official/docs/node/current/users/retrieve.js]: Whoops 😂 Fixed

**Review by jchen293 — APPROVED** [review]
Nice work! Approving this now but we should not merge until the Node V6 is released.

**Review by nwithan8 — APPROVED** [review]

**Comment by Justintime50** [comment]
Drafting till we release v6 shortly

**Pull request merged**